frontend/src/components/climateMatchResults/ClimateMatchSuggestionInfo.js

```diff
@@ -171,7 +171,11 @@ export default function ClimateMatchSuggestionInfo({
             handleClickContact={handleClickContact}
             explanationBackground={background}
             className={classes.contactCreatorButton}
-            large={isNarrowScreen}
+            withStartIcon={!isNarrowScreen}
+            withAvatar={!isNarrowScreen}
```

Comment:
Some of these properties seem to be always the same (at least withAvatar and withStartIcon), maybe we can consider merging them. I understand the idea of reusability but I think it's good to avoid including more props than needed.

Reply:
Good point! I merged withAvatar and withStartIcon into one prop (withIcons) :)